crates/aptos/src/move_tool/fmt.rs

```diff
@@ -127,31 +127,39 @@ impl FmtCommand {
             for file in &files {
                 if !file.exists() {
                     return Err(CliError::UnexpectedError(format!(
-                        "File does not exist:{:?}",
-                        file
+                        "File does not exist:{}",
+                        file.display()
                     )));
                 }
                 if !file.is_file() {
                     return Err(CliError::UnexpectedError(format!(
-                        "Path is not a file:{:?}",
-                        file
+                        "Path is not a file:{}",
+                        file.display()
                     )));
                 }
                 if file.extension().unwrap() != move_command_line_common::files::MOVE_EXTENSION {
                     return Err(CliError::UnexpectedError(format!(
-                        "File is not a Move file:{:?}",
-                        file
+                        "File is not a Move file:{}",
+                        file.display()
                     )));
                 }
             }
             files
         } else {
             // Handle package path
-            let package_opt = if let Some(path) = package_opt {
-                fs::canonicalize(path.as_path()).ok()
-            } else {
-                None
+            let package_opt = match package_opt {
+                Some(path) => {
+                    let abs = fs::canonicalize(path.as_path()).map_err(|_| {
+                        CliError::UnexpectedError(format!(
+                            "Specified path {} does not exist",
```

Comment:
```suggestion
                            "Specified package path {} does not exist",
```

Reply:
I think path makes more sense here. This is only caught if path isn't on file system. If path does exist but isn't a package, this is instead caught by the error `Unexpected error: Unable to find package manifest in [valid_path] or in its parents`